sympy/simplify/hyperexpand.py

```diff
@@ -2417,12 +2427,16 @@ def weight(expr, cond):
     return func0(z0)
 
 
-def hyperexpand(f, allow_hyper=False, rewrite='default'):
+def hyperexpand(f, allow_hyper=False, rewrite='default', place=None):
     """
     Expand hypergeometric functions. If allow_hyper is True, allow partial
     simplification (that is a result different from input,
     but still containing hypergeometric functions).
 
+    If a G-function has expansions both at zero and at infinity,
+    ``place`` can be set to ``0`` or ``zoo`` to indicate the
+    preferred choice.
```

Comment:
If there are expansions both at zero and at infinity, hyperexpand first tries to find closed form representations for them, and if it succeeds in both cases, the default is the one with smaller number of operations.